styles.css

```diff
@@ -0,0 +1,122 @@
+body{
+    font-family: 'Nunito', sans-serif;
+    background: #FFB75E;  /* fallback for old browsers */
+    background: -webkit-linear-gradient(to right, #ED8F03, #FFB75E);  /* Chrome 10-25, Safari 5.1-6 */
+    background: linear-gradient(to right, #ED8F03, #FFB75E); /* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */
+}
+
+:root {
+    --primary-color: #0ea5e9;
+    --stone: #78716c;
+    --background: #d1d5db;
+    --orange: #f97316;
```

Comment:
avoid using color names as css variables, if you plan to change it later then you need to change the name too, kills the purpose of using a variable. Use generic names